src/subdomains/supporting/payment/dto/tx-fee-details.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { FeeDto } from './fee.dto';
+
+export interface TxFeeDetails extends FeeDto {
+  minFee: number;
+  minVolume: number;
+  feeAmount: number;
+}
```

Comment:
It's a bit weird to have `fixed` and `rate` (w/o fee), but also `minFee` (w/ fee) in the same DTO?